Browser/browser.py

```diff
@@ -599,6 +601,7 @@ def __init__(
         run_on_failure: str = "Take Screenshot",
         external_browser_executable: Optional[Dict[SupportedBrowsers, str]] = None,
         jsextension: Optional[str] = None,
+        enable_presenter_mode: bool = False,
```

Comment:
Instead of bool value, this could be better  be : `enable_presenter_mode: Optional[int] = None`
Then one could use the value to control how long the element highlighted.  Example like:
```
        if self.library.presenter_mode:
            self.hover(selector)
            self.library.highlight_elements(selector, duration=timedelta(seconds=self.library.presenter_mode))
```
And not to hard code the timeout in some fixed value.

Reply:
Hard coded value is easier to use. More thinking on our side with sensible default.